census_view.cpp

```diff
@@ -1110,6 +2088,41 @@ wxWindow* CensusDVCView::CreateChildWindow()
     return list_window_;
 }
 
+wxWindow* CensusGridView::CreateChildWindow()
+{
+    grid_window_ = new(wx) wxGrid(GetFrame(), wxID_ANY);
+    grid_window_->HideRowLabels();
+    grid_window_->SetColLabelAlignment(wxALIGN_LEFT, wxALIGN_CENTRE);
+    grid_window_->SetDefaultCellAlignment(wxALIGN_LEFT, wxALIGN_CENTRE);
+    grid_window_->SetDefaultCellFitMode(wxGridFitMode::Ellipsize(wxELLIPSIZE_MIDDLE));
+    grid_window_->SetGridLineColour(wxSystemSettings::GetColour(wxSYS_COLOUR_BTNFACE));
```

Comment:
I think this should be done in wx itself by default, hardcoding `0xc0c0c0` there as is done now is clearly not right. I'll fix this there and remove this line when it's done.

Reply:
Done in wx commit f1425dad13 (Don't hard code grid lines colour in wxGrid, 2020-05-23)